### DISCUSSION TITLE
[Question] Tile as a Card component?

### DISCUSSION THREAD

**Discussion by mrcasablr** [carbon-design-system/carbon-components-svelte#1319] [General]
Hello,

Thank you for maintaining the svelte components for this design system. This has helped me build many small projects. I often run into building a Card for such projects and I understand upstream does not have an out of the box Card component available - https://carbondesignsystem.com/components/tile/usage/ and recommends using a Tile component for it.

Anyone can share some examples on how they are getting around to this?

Thanks in advance!

**Comment by metonym** [comment]
Tile docs: https://svelte.carbondesignsystem.com/components/Tile

Inside, you can use [Stack ](https://svelte.carbondesignsystem.com/components/Stack)to compose elements using flex layouts.
